src/lib/renderer/renderer.tsx

```diff
@@ -111,14 +126,14 @@ export interface IRenderComponentProps {
  * allows for specifying components.
  *
  * @see  {RenderForm} for more information.
- * @external {CallbacksContext} Expects `CallbackContext` to be available.
+ * @external {FormikContext} Expects `Formik`/`FormikContext` to be available.
  * @external {RenderContext} Expects `RenderContext` to be available.
- * @external {SubmissionContext} Expects `SubmissionContext` to be available.
  */
 export const RenderComponent = ({component}: IRenderComponentProps): React.ReactElement => {
-  const callbacks = useContext(CallbacksContext);
-  const value = useValue(component);
   const Component = useComponentType(component);
+  const field = useField(component.key || OF_MISSING_KEY);
+  const {value, onBlur, onChange} = field[0];
+  const callbacks = {onBlur, onChange};
```

Comment:
```suggestion
  const [{value, onBlur, onChange}] = useField(component.key || OF_MISSING_KEY);
  const callbacks = {onBlur, onChange};
```
The args `meta` and `helpers` are not relevant at the moment, but they might become.

IMO this makes it a bit more obvious that the callbacks originate from formik and are re-wrapped for our own purposes.

I'm also not sure if we should pass `callbacks` as an object to the component, or just the callbacks themselves, but that'll become clearer later on.